diff --git a/WORKSPACE b/WORKSPACE
index 683cc7e..69cca13 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -9,6 +9,7 @@ http_archive(
     workspace_file_content = "//ml_metadata:postgresql.WORKSPACE",
     sha256 = "9868c1149a04bae1131533c5cbd1c46f9c077f834f6147abaef8791a7c91b1a1",
     strip_prefix = "postgresql-12.1",
+    patches = ["//ml_metadata/third_party:postgresql.patch",],
     urls = [
         "https://ftp.postgresql.org/pub/source/v12.1/postgresql-12.1.tar.gz",
     ],
